pca_tools/check_couch.py

```diff
@@ -0,0 +1,45 @@
+import os
+import couchdb
+import getpass
+
+# connect to couch
+pw = getpass.getpass("CouchDB password: ")
+couchserver = couchdb.Server("http://snoplus:%s@couch.snopl.us" % pw)
```

Comment:
There should perhaps be a config file from which these hard-coded settings are loaded. Compare to tellie.cfg in the base directory...

Reply:
taken to config